.github/ISSUE_TEMPLATE/integration_bug.yml

```
@@ -293,6 +293,12 @@ body:
 
     validations:
       required: true
+  - type: input
+    id: datastream_name
+    attributes:
+      label: Data Stream Name
+      description: Which data stream in the integration package are you having issues with?
+      placeholder: ex. logs
```

Are we able to use markdown in the description? We could link out to https://www.elastic.co/guide/en/fleet/current/data-streams.html for more context. If we ask for the dataset name then we can get the submitter to reaffirm the integration name too. e.g.

```suggestion
    id: data_stream_dataset_name
    attributes:
      label: Dataset Name
      description: Which [dataset](https://www.elastic.co/guide/en/fleet/current/data-streams.html) in the integration package are you having issues with?
      placeholder: ex. nginx.access
```

---

Let's try.

---

Yes, that works.